rust/src/smb/smb.rs

```diff
@@ -1167,8 +1197,12 @@ impl SMBState {
                     SCLogDebug!("post_gap_housekeeping: done");
                     break;
                 }
-                if let Some(SMBTransactionTypeData::FILE(_)) = tx.type_data {
+                if let Some(SMBTransactionTypeData::FILE(ref mut f)) = tx.type_data {
                     // leaving FILE txs open as they can deal with gaps.
+                    if f.post_gap_ts != 0 {
+                        f.post_gap_ts = self.ts + 60;
+                        self.check_post_gap_file_txs = true;
+                    }
```

Comment:
Something I'm not following is how does f.post_gap_ts ever not != 0, I only see it being set if not already 0.

Reply:
Whoops ya seeing the issue. Will submit a new PR. Thanks!